modules/mono/mono_gd/gd_mono_internals.cpp

```diff
@@ -30,6 +30,8 @@
 
 #include "gd_mono_internals.h"
 
+#include "core/os/os.h"
+
```

Comment:
I guess this include is no longer needed?

Reply:
Apparently not! It's removed now.